### ISSUE TITLE
Particle crash

### ISSUE THREAD

**Issue by Divineaspect** [Esteemed-Innovation/Flaxbeards-Steam-Power#349]
http://pastebin.com/vKf5b7RG

So this was pretty interesting. I had set up a a few fans pointing at a thaumcraft crimson cultist spawner, with a few vacuums on the other side, with the intention of making a monster grinder.

Using the wrench to change one of the fan's range either to or past 15 seems to have triggered the crash. the vaccums were 16 blocks away in line with the fans.

**Comment by elifoster** [comment]
I don't think this is crossmod; I think it's just a result of the following code:
``` java
public void spawnBreakParticles(World world, float x, float y, float z, Block block, float xv, float yv, float zv) {
        if (!Config.disableParticles) {
            Minecraft.getMinecraft().effectRenderer.addEffect((new EntityDiggingFX(world, x, y, z, xv, yv, zv, block, 0)));
        }
    }
```

being called on a `block` that does not have digging effects.
